docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -15,53 +15,63 @@ use the following commands to launch the placement translator; it
 requires the ESPW to be current at start time.
 
 ```shell
-# TODO: placement-translator needs access to multiple configs. Will remove when controllers support spaces.
-kubectl ws root:espw
-placement-translator &
-sleep 10
+(
+  export KUBECONFIG=$SM_CONFIG
```

Comment:
This is not consistent with what I see in cmd/placement-translator/main.go . That code is still consuming three kube client configs, for various kcp things.